t/full-stack.t

```diff
@@ -160,6 +160,13 @@ sub bail_with_log ($job_id, $message) {
     # uncoverable statement count:2
     my $log = eval { $log_file->slurp };
     diag $@ ? "unable to read $log_file: $@" : "$log_file:\n$log";    # uncoverable statement
+                                                                      # uncoverable subroutine
```

Comment:
why that?

Reply:
I copied the existing code including all comments.

Reply:
Let's see what happens if I drop those two "stray" comments. Maybe doesn't mean what I thought it means